lib/py/setup.py

```diff
@@ -22,7 +22,7 @@
 import sys
 try:
     from setuptools import setup, Extension
-except:
+except Exception:
```

Comment:
I think we should use distutils.core, and setuptools is feedback,
And we should only process ImportError.
```py
try:
     from distutils.core import setup, Extension
except ImportError:
    from setuptools import setup, Extension
```

Reply:
I have to resubmit anyway to kick off another PR build, but I'd like to leave it as I changed it for now, and it can be improved later through a subsequent PR.  I was just trying to eliminate those pesky E722 errors in flake8.  There are a lot more issues in generated code that flake8 pulls out.